apps/cardio-patient/src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Charts-on-FHIR Cardiovascular Health</title>
+    <title>Charts-on-FHIR Cardio Patient</title>
```

Comment:
Let's keep the original app title here

Reply:
Can we use "Cardiovascular Patient App" as the title for cardio-patient?